<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreGUI::IOSettingsEditorPage</class>
 <widget class="QWidget" name="CoreGUI::IOSettingsEditorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input/Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Terminal width</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="useFixedWidth">
        <property name="text">
         <string>Use fixed width for new launch sessions</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Width in characters:</string>
          </property>
          <property name="buddy">
           <cstring>widthInCharacters</cstring>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="widthInCharacters">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>184</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>useFixedWidth</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useFixedWidth</sender>
   <signal>toggled(bool)</signal>
   <receiver>widthInCharacters</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
